packages/@aws-cdk/aws-apigatewayv2/lib/http/route.ts

```diff
@@ -19,6 +20,22 @@ export interface IHttpRoute extends IRoute {
    * Returns the path component of this HTTP route, `undefined` if the path is the catch-all route.
    */
   readonly path?: string;
+
+  /**
+   * Grant access to invoke the route.
+   */
+  grantInvoke(grantee: iam.IGrantable, options: GrantInvokeOptions | undefined): iam.Grant;
+}
+
+/**
+ * Options for granting invoke access.
+ */
+export interface GrantInvokeOptions {
+  /**
+   * The HTTP method to allow.
+   * @default `HttpMethod.ANY`
+   */
+  readonly httpMethod?: HttpMethod;
```

Comment:
Feels like this should be a list.

Reply:
Ok. I've made this a list. Now elements of the list map to multiple resources in a single grant statement.